.gitignore

```diff
@@ -37,3 +37,5 @@ node_modules
 /test/output/*.bmp
 
 .DS_Store
+.settings
+/update_from_upstream.bat
```

Comment:
this seems like a script that's particular to your computer. Can we remove this line?